src/import.cmd.js

```diff
@@ -36,7 +37,20 @@ export default class ImportCommand extends AbstractServerCommand {
   }
 
   withUIRepo(value) {
-    this._uiRepo = value;
+    this._uiBranch = GitUtils.DEFAULT_BRANCH;
+    try {
+      const url = new URL(value);
+      if (url.hash) {
+        this._uiBranch = url.hash.substring(1);
+      }
+      // Ensure a dangling hash does not create an invalid uiRepo.
+      url.hash = '';
+      this._uiRepo = url.href;
```

Comment:
☝️ Always remove hash and set uiRepo.  This also covers the case where `value` has a dangling hash, making the url invalid for future 'git' commands:
`https://github.com/adobe/helix-importer-ui#`